engine/demo/src/test/resources/application.conf

```diff
@@ -1,4 +1,9 @@
-checkpointInterval: 10s
+checkpointConfig {
+    checkpointInterval: 10s
```

Comment:
isn't checkpoint in all these properties necessary? they're in checkpointConfig anywa? maybe interval, maxConcurrent etc. ?

Reply:
I'd prefer to keep long names, because flink has them in such format